CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# Release 2.20.0 05-11-2020
+
+- ConfigMgr changes: Jwt_authn use per-route config, remove path_matcher (#403)
+- Add config generator option to allow startup on errors (#402)
```

Comment:
#402 is not hooked up yet.  we should list it in the next release

Reply:
+1

Reply:
Done.